src/C3D/Extensions/Aspire/IISExpress/ApplicationHostConfiguration/ApplicationHostConfiguration.cs

```diff
@@ -1,10 +1,868 @@
-﻿using System.Xml.Serialization;
+﻿
+using EnvDTE90;
+using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+#nullable disable
 
 namespace C3D.Extensions.Aspire.IISExpress.Configuration;
 
```

Comment:
Is this auto generated, or is it from 'somewhere' that has the full structure of this file?
I doubt much is likely to be added to IIS / IISExpress, but what happens if this changes in the future?

Reply:
I took the template file and ran it through a tool online. I'll dig it up and put a comment as to how to regenerate it. I found the "copy xml as classes" generates really ugly code and the website was fairly clean.